blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMetricDataQueryServer.java

```diff
@@ -79,6 +80,10 @@ public void startServer() {
                         Executors.newFixedThreadPool(workerThreads)));
         server.setPipelineFactory(new MetricsHttpServerPipelineFactory(router));
         serverChannel =  (ServerChannel) server.bind(new InetSocketAddress(httpQueryHost, httpQueryPort));
+
+        //register the tracker MBean for JMX/jolokia
+        log.info("Registering tracker service");
+        Tracker.getInstance();
     }
```

Comment:
this was why I added the register() call earlier, because of the behavior I was seeing.  Maybe if I add it back the code will read better.  Unless you know of a way to define Tracker so it does call the private constructor non-lazily?